canonical-web-and-design/juju.is
# Build the about universal-operators page

## QA
- Go to http://0.0.0.0:8041/about/universal-operators
- Check the content matches [the copy doc](https://docs.google.com/document/d/1vdT7mA2GlXxad12VbuIx12QtNktbKkjWnkcZd7LJo4A/edit#)


Fixes https://github.com/canonical-web-and-design/juju.is/issues/101
